src/opml-generator.js

```diff
@@ -0,0 +1,19 @@
+const opml = require('opml-generator');
```

Comment:
I would move all this code into your opml.js route.  I don't think there's much value breaking it out here.